InfiniteRecharge/src/main/java/frc/robot/Constants.java

```diff
@@ -23,6 +23,21 @@
 
 	// Robot Wrangler
 	public static final boolean kUseRobotWranglerNeoEncoder = true;
+	public static final double kIndexerSpeed = 1;
+
+	public static final int kIntakeCanID = 9;
+	public static final int kIntakeActuatorCanID = 10;
+
+	public static final int kDriveCurrentLimit = 40;
+
+	public static double kShooterP = 6e-5; 
+    public static double kShooterI = 0;
+    public static double kShooterD = 0; 
+    public static double kShooterIz = 0; 
+	public static double kShooterFF = 0.000015; 
```

Comment:
what did you do to get this number?

Reply:
It is from the spark max velocity pid examples.